.github/workflows/release_production.yml

```diff
@@ -53,71 +56,65 @@ jobs:
       id: extract_version
       run: echo "RELEASE_VERSION=${version}" >> $GITHUB_OUTPUT
     - name: Publish to Cloudflare Pages Production
+      id: cloudflare-upload
       uses: "./.github/actions/publish_to_pages_production"
       with:
         CLOUDFLARE_ACCOUNT_ID: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
         CLOUDFLARE_API_TOKEN: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+      continue-on-error: true
+    - name: Upload to vercel
+      id: vercel-upload
+      uses: 'deriv-com/shared-actions/.github/actions/vercel_DR_publish@master'
+      with:
+        VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+        VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+        VERCEL_TOKEN: ${{ secrets.VERCEL_API_TOKEN }}
+        ENVIRONMENT: Production
+        VERCEL_SCOPE: deriv
+        ALIAS_DOMAIN_URL: 'app-dr.deriv.com'
+      continue-on-error: true
     - name: Upload Build Artifact
+      id: artifact-upload
       uses: actions/upload-artifact@26f96dfa697d77e81fd5907df203aa23a56210a8
       with:
         name: build
         path: packages/core/dist
         retention-days: 1
+      continue-on-error: true
+    - name: Check for failures
+      id: check-failures
```

Comment:
I dont understand how does this work!
why we need to have this?
 By right we should have two different slack notification step separated from each other.
 One for production release.
 One for DR solution.
 In any case we need to send the production slack notif, I mean both success or failure.
 For DR we just send the slack notif if it fails.

Reply:
Will make the change and push